_assets/ci/get-docker-image-tag.sh

```diff
@@ -1,18 +1,9 @@
 #!/usr/bin/env bash
 #
-# Creates a string in a format: $GIT_SHA[:8][-$BUILD_TAGS]
-# where $BUILD_TAGS is optional and if present all spaces
-# are replaced by a hyphen (-).
-#
-# For example: BUILD_TAGS="tag1 tag2" ./_assets/ci/get-docker-image-tag.sh
-# will produce "12345678-tag1-tag2".
+# Returns a tag for a docker image. It tries to get a tag first if availalbe,
+# otherwise the first 8 characters of the HEAD git SHA is returned.
 
 set -e -o pipefail
 
-tag="$(git rev-parse HEAD | cut -c 1-8)"
-
-if [ ! -z "$BUILD_TAGS" ]; then
-    tag="$tag-$(echo $BUILD_TAGS | sed -e "s/[[:space:]]/-/g")"
-fi
-
+tag="$(git describe --always --tag || git rev-parse HEAD | cut -c 1-8)"
```

Comment:
Actually, I made a mistake. What I wanted to do is `git describe --exact-match --tag` which would fail if there is no exact match of a tag for the current HEAD. In such a case it will fall back to the old solution with 8 digits of SHA.